common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/BlockTransferMessage.java

```diff
@@ -35,6 +35,9 @@
  *     shuffle service. It returns a StreamHandle.
  *   - UploadBlock is only handled by the NettyBlockTransferService.
  *   - RegisterExecutor is only handled by the external shuffle service.
+ *   - StreamHandle is only handled by the OneForOneBlockFetcher.
```

Comment:
This is already covered in the documentation of `OpenBlock`, since `StreamHandle` is the reply to `OpenBlock`.